full-v3.yml

```diff
@@ -0,0 +1,2299 @@
+          schema:
+            $ref: '#/definitions/SectionsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /terms/{id}/schools:
+    get:
+      tags: ["Terms"]
+      description: Returns the schools for a term
+      operationId: getSchoolsForTerm
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users:
+    get:
+      tags: ["Users"]
+      description: Returns a list of users
+      operationId: getUsers
+      parameters:
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      - in: query
+        name: count
+        type: string
+        enum:
+        - ""
+        - "true"
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/UsersResponse'
+
+  /users/{id}:
+    get:
+      tags: ["Users"]
+      description: Returns a specific user
+      operationId: getUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/UserResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/district:
+    get:
+      tags: ["Users"]
+      description: Returns the district for a user
+      operationId: getDistrictForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/mycontacts:
+    get:
+      tags: ["Users"]
+      description: Returns the contacts for a user
+      operationId: getContactsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/ContactsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/mystudents:
+    get:
+      tags: ["Users"]
+      description: Returns the students for a user
```

Comment:
Not sure if it's necessary to write down and/or where to put it, but the user ID has to be a teacher's.